.github/workflows/ci.yml

```
@@ -116,7 +116,7 @@ jobs:
         id: image-name
 
   open-api-ci:
-    uses: maykinmedia/open-api-workflows/.github/workflows/ci.yml@v5
+    uses: maykinmedia/open-api-workflows/.github/workflows/ci.yml@refactor/replace-check-sphinx
```

Updated v5 with the changes from your PR
```suggestion
    uses: maykinmedia/open-api-workflows/.github/workflows/ci.yml@v5
```